vertical-pod-autoscaler/e2e/v1/common.go

```diff
@@ -257,47 +288,32 @@ func SetupHamsterContainer(cpu, memory string) apiv1.Container {
 
 // SetupVPA creates and installs a simple hamster VPA for e2e test purposes.
 func SetupVPA(f *framework.Framework, cpu string, mode vpa_types.UpdateMode, targetRef *autoscaling.CrossVersionObjectReference) {
-	vpaCRD := NewVPA(f, "hamster-vpa", targetRef)
-	vpaCRD.Spec.UpdatePolicy.UpdateMode = &mode
-
-	cpuQuantity := ParseQuantityOrDie(cpu)
-	resourceList := apiv1.ResourceList{apiv1.ResourceCPU: cpuQuantity}
-
-	vpaCRD.Status.Recommendation = &vpa_types.RecommendedPodResources{
-		ContainerRecommendations: []vpa_types.RecommendedContainerResources{{
-			ContainerName: "hamster",
-			Target:        resourceList,
-			LowerBound:    resourceList,
-			UpperBound:    resourceList,
-		}},
-	}
-	InstallVPA(f, vpaCRD)
+	SetupVPAForNHamsters(f, 1, cpu, mode, targetRef)
```

Comment:
Please annotate that 1 is number of hamsters

Reply:
/lgtm

overall but I would like to see argumments marked with names for easier reasding

Reply:
I would prefer to let the args without annotations here. The signature of the called function is three lines below...